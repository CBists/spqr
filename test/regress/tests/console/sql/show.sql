SHOW shards;
SHOW routers;
SHOW sharding_rules;
SHOW key_ranges;
