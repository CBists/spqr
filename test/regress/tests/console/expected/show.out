
		SQPR router admin console
	Here you can configure your routing rules
------------------------------------------------
	You can find documentation here 
https://github.com/pg-sharding/spqr/tree/master/docs

SHOW shards;
  listing data shards  
-----------------------
 datashard with ID sh1
 datashard with ID sh2
(2 rows)

SHOW routers;
 show routers  
---------------
 router local-
(1 row)

SHOW sharding_rules;
          listing sharding rules          
------------------------------------------
 sharding rule r1 with column set: [w_id]
 sharding rule r2 with column set: [id]
 sharding rule r3 with column set: [bid]
(3 rows)

SHOW key_ranges;
 Key range ID | Shard ID | Lower bound | Upper bound 
--------------+----------+-------------+-------------
 krid1        | sh1      | 1           | 10
 krid2        | sh1      | 11          | 20
(2 rows)

